src/core/ad.jl

```diff
@@ -433,3 +475,53 @@ MvNormal(m::TrackedVector{<:Real}, A::UniformScaling{<:Real}) = MvNormal(m, A.λ
 
 # zero mean,, constant variance
 MvNormal(d::Int, σ::TrackedReal{<:Real}) = MvNormal(zeros(d), σ)
+
+import Bijectors: link, invlink
+using Bijectors: Dirichlet, PDMatDistribution
+
+for F in (:link, :invlink)
+    @eval begin
+        function $F(
+            dist::Dirichlet, 
+            x::Tracker.TrackedArray, 
+            ::Type{Val{proj}} = Val{true}
+        ) where {proj}
+            return Tracker.track($F, dist, x, Val{proj})
+        end
+        Tracker.@grad function $F(
+            dist::Dirichlet, 
+            x::Tracker.TrackedArray, 
+            ::Type{Val{proj}}
+        ) where {proj}
+            x_data = Tracker.data(x)
+            T = eltype(x_data)
+            y = $F(dist, x_data, Val{proj})
+            return  y, Δ -> begin
+                out = (ForwardDiff.jacobian(x -> $F(dist, x, Val{proj}), x_data)::Matrix{T})' * Δ
```

Comment:
Is this a duplicate fix for https://github.com/TuringLang/Bijectors.jl/pull/28?

Reply:
@mohamed82008

Reply:
No we won't need `ForwardDiff`, we can replace it with the `link_jacobian` or `invlink_jacobian` functions.

Reply:
Do you want to remove these functions from this PR, or keep them until TuringLang/Bijectors.jl#28 is merged?

Reply:
I would keep them for now as a stop gap. This fixes the problem but can be improved when the Bijectors PR is merged.

Reply:
We are depending on ForwardDiff anyways so nothing to lose.